modules/virt-vmware-comparison.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/about_virt/about-virt.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="virt-vmware-comparison_{context}"]
+= Comparison of {VirtProductName} with VMware vSphere
+
+{VirtProductName} is conceptually different from VMware vSphere, much of its functionality comes from the underlying {product-title}, and it therefore does not offer direct alternatives to individual vSphere concepts and components. That said, if you are familiar with VMware vSphere, the following table lists the most common vSphere concepts and outlines what {VirtProductName} components you can use to accomplish a similar task.
+
+.Mapping of vSphere concepts to their closest {VirtProductName} counterparts
+[options="header"]
+[cols="2,2,3"]
+|===
+|vSphere concept |{VirtProductName} |Explanation
+
+|Datastore
+|Persistent volume (PV){nbsp}+ +
+Persistent volume claim (PVC)
+|Stores VM disks. A persistent volume represents existing storage and is attached to a VM through a persistent volume claim. When created with the `ReadWriteMany` (RWX) access mode, persistent volume claims can be mounted by multiple VMs simultaneously.
+
+|Dynamic Resource Scheduling (DRS)
+|Pod eviction policy{nbsp}+ +
+Descheduler
+|Provides active resource balancing. A combination of pod eviction policies and a descheduler allows VMs to be live migrated to more appropriate nodes to keep node resource utilization manageable.
+
+|NSX
+|Multus{nbsp}+ +
+OVN-Kubernates{nbsp}+ +
+Third-party container network interface (CNI) plug-ins
+|Provides overlay network configuration. There is no direct equivalent for NSX in {VirtProductName}, depending on the desired functionality, users can either rely on the OVN-Kubernates network provider, or install certified third-party CNI plug-ins.
+
+|Storage Policy Based Management (SPBM)
+|Storage class
+|Provides policy-based storage selection. Storage classes represent various storage types and describe storage capabilities such as quality of service, backup policy, as well as the reclaim policy or whether volume expansion is allowed. Persistent volume claims can request a specific storage class to satisfy the application requirements.
```

Comment:
This needs a little editing but is almost there. I don't think this suggestion is perfect, but perhaps heading in the right direction:

```suggestion
|Provides policy-based storage selection. Storage classes represent various storage types and describe storage capabilities, such as quality of service, backup policy, reclaim policy, and whether volume expansion is allowed. A PVC can request a specific storage class to satisfy application requirements.
```